docs/book/intro.md

```diff
@@ -82,7 +82,7 @@ $twitter = new Laminas\Twitter\Twitter($config);
 Make sure you substitute the values you discovered earlier in the
 configuration before attempting to connect.
 
-> #### Note
+> ### Note on SSL certificate verification
```

Comment:
For headlines we use the Chicago Manual of Style:

```suggestion
> ### Note on SSL Certificate Verification
```
This online tool can help: https://capitalizemytitle.com/style/Chicago/